layers/+lang/markdown/README.org

```diff
@@ -187,6 +187,7 @@ To generate a table of contents type on top of the buffer:
 | ~SPC m P~   | previous link      |
 
 ** Promotion, Demotion
+These keybindings only work in evil-normal and evil-insert modes.
```

Comment:
Do you think we should specify here that they are avaiable under `C-c C-x` for holy mode? These are set by the markdown mode so we don't have control over when they change but I feel it's unlikely it will.